### ISSUE TITLE
No files load but able to tab on title of Data Plot section

### ISSUE THREAD

**Issue by baozi90** [RITAccess/data-analyzer#28]
Should not be able to tab on the Data Plot title when no file is loaded.

**Comment by smcintosh881** [comment]
The black VoiceOver box doesn't highlight anything on my screen but it stills reads "Data Points group"
